week-2/Homework/mandatory/3-playing-computer.js

```diff
@@ -12,12 +12,12 @@
 
   Answer the following questions:
 
-  1. This program throws an error. Why? (If you can't find it, try executing it).
-  2. Remove the line that throws the error.
-  3. What is printed to the console?
-  4. How many times is "f1" called?
-  5. How many times is "f2" called?
-  6. What value does the "a" parameter take in the first "f1" call?
+  1. This program throws an error. Why? (If you can't find it, try executing it). VAR B HAS NOT BEEN DEFINED. 
+  2. Remove the line that throws the error. REMOVED: console.log(b); 
+  3. What is printed to the console? 2 6
+  4. How many times is "f1" called? 4 TIMES
+  5. How many times is "f2" called? 2 TIMES 
+  6. What value does the "a" parameter take in the first "f1" call? // I HAVE TRIED HARD BUT CANNOT UNDERSTAND THE LAST 2 QUESTIONS, CAN SOMEONE EXPLAIN PLEASE?  
```

Comment:
Please ask your assigned mentors to demystify this in live coding call. It'd be way faster and easier to explain and understand instead of writing a novel here.